src/app/cpe/project/project.component.html

```diff
@@ -0,0 +1,19 @@
+<div *ngIf="project">
+  <div [id]="project.id" style="border: blue solid 2px; height: 70vh; width: 90%;">
+    <div style="font-weight: bold; margin: 10px;">{{project.label}}</div>
+    <div style="margin: 10px">{{project.description}}</div>
+    <div style="margin: 10px">
+      <h2 style="font-weight: bold">It has pages:</h2>
+      <div style="display: flex; flex-direction: row">
+        <div *ngFor="let page of project.pages" style="margin: 10px">
```

Comment:
normally styles should be defined in the css or scss file not inline

Reply:
Wrong in this and our case.  This is a mock component, this is about routing. The decision to make it inline was simply to not pollute or use any scss if the component gets removed or rewritten. I do not wan't to have any scss files in my repository containing sth. which might be used or not. Furthermore we are using tailwind css - no scss. Inline styles are the best option there. In this very case and concerning future use of tw one simply does not want to use any scss. Does our UX have any styles defined yet? C'mon please.